packages/features/ee/workflows/lib/reminders/utils.ts

```diff
@@ -0,0 +1,35 @@
+import { dub } from "@calcom/features/auth/lib/dub";
+
+export type EventLinks = {
+  meetingUrl: string;
+  cancelLink: string;
+  rescheduleLink: string;
+  ratingUrl: string;
+  noShowUrl: string;
+};
+
+export const getShortenLinks = async (longLinks: EventLinks) => {
+  const shortLinks: EventLinks = {
+    meetingUrl: "",
+    cancelLink: "",
+    rescheduleLink: "",
+    ratingUrl: "",
+    noShowUrl: "",
+  };
+
+  const linkTypes = Object.keys(longLinks);
+
+  for (const link of linkTypes) {
+    const linkType = link as keyof EventLinks;
+    const url = longLinks[linkType];
+
+    if (url) {
+      const { shortLink } = await dub.links.create({
```

Comment:
Does it make sense to Promise.allSettled this and fire all at the same time?

Reply:
@emrysal you can use `dub.links.createMany` to create them all at once: https://dub.co/docs/sdks/typescript/links/create-many

To define `sms.cal.com` as the domain, you can do pass that as an attribute in the body payload:

```json
{
		domain: "sms.cal.com",
		url,
}
```

Reply:
@VK-RED can you add this? ^

Reply:
Sure @PeerRich will do the changes, also should I add the domain in .env.example also ?

Reply:
@emrysal Used Promise.allSettled was thinking of dub.links.createMany, but as we can also have empty links , we dont want to hit dub with the empty ones..